Disable CUDA explicitly
Based on original patch by Henrik Nagel

--- fsl/config/buildSettings.mk	2021-06-11 16:19:38.000000000 +0100
+++ fsl.new/config/buildSettings.mk	2021-11-16 12:10:42.276368838 +0000
@@ -182,8 +182,8 @@
 #$(info GCC PAT VER $(GCCPAT))
 #Project specific variables
 EDDYBUILDPARAMETERS = "cuda=1 CUDAVER=8.0" "cuda=1 CUDAVER=9.1" "cuda=1 CUDAVER=10.2" "cpu=1"
-fdt_MASTERBUILD     = COMPILE_GPU = 1
-ptx2_MASTERBUILD    = COMPILE_GPU = 1
+fdt_MASTERBUILD     = COMPILE_GPU = 0
+ptx2_MASTERBUILD    = COMPILE_GPU = 0
 define newline
 
 
